src/pages/Quote.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Quote() {
+  return (
+    <h1>
+      {' '}
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis
+    </h1>
+  );
+}
```

Comment:
- [ ]  You can make your website more beautiful as it attracts the recruiter and helps in your job search. You can add your own styles to improve the look and feel but you must stick to the general layout that matches the given wireframes.
- [ ] There is a heading on the left of the page and content after that. Also, see the spacing and please implement it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213714343-bac224bf-b64c-4272-847b-58f4ded908c4.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/213714363-b256c76a-3522-4d87-b312-35702d8e4b15.jpg)